Workbooks/WorkbooksMetadata.json

```diff
@@ -1415,5 +1415,18 @@
       "templateRelativePath": "AzureSentinelCost.json",
       "subtitle": "",
       "provider": "Azure Sentinel Community"
-    }   
+    },   
+    {
+      "workbookKey": "AzurePurview",
+      "logoFileName": "AzurePurview.svg",
+      "description": "This workbook provides insights into the sensitivity of data that has been scanned by Azure Purview, including classifications and labels found across the data estate.",
+      "dataTypesDependencies": ["PurviewDataSensitivityLogs"],
+      "dataConnectorsDependencies": ["AzurePurview"],
```

Comment:
Please change the data connector id to : "MicrosoftAzurePurview", it should match the id property in the data connector.